src/2.2-counter-task/Counter.jsx

```diff
@@ -1,16 +1,16 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
-export default function Counter() {
-  const [count, setCount] = useState(0);
+export default function Counter({ initialValue, min, max }) {
+  const [count, setCount] = useState(initialValue ? initialValue : 0);
```

Comment:
You can just set a default value for prop:

```jsx
function Counter({ initialValue = 0, min, max }) {
  const [count, setCount] = useState(initialValue);
```

Might be a bit easier to understand